src/main/java/wolox/training/repositories/BookRepository.java

```diff
@@ -0,0 +1,11 @@
+package wolox.training.repositories;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+import wolox.training.models.Book;
+
+@Repository
+public interface BookRepository extends JpaRepository<Book, Long> {
+
+    Book findByAuthor(String author);
```

Comment:
The book should be wrapped around an **Optional** so as to avoid nullchecks in the future, check this out:  
https://www.baeldung.com/java-optional 
Let us know if you have any doubts about this :)